superset-frontend/spec/javascripts/sqllab/ResultSet_spec.jsx

```diff
@@ -182,3 +183,18 @@ describe('ResultSet', () => {
     });
   });
 });
+
+describe('RTL ResultSet tests', () => {
+  it('renders if there props has no limit in query.results but has queryLimit', () => {
+    render(<ResultSet {...mockedProps} />, { useRedux: true });
+    const grid = screen.getAllByRole('grid');
+    expect(grid.length).toBe(1);
```

Comment:
If there is only one grid expected, then we can use `getBy` instead of `getByAll`.
```suggestion
    expect(screen.getByRole('grid')).toBeInTheDocument());
```